src/ucs/debug/log.h

```diff
@@ -75,21 +86,23 @@ typedef enum {
     UCS_LOG_FUNC_RC_CONTINUE
 } ucs_log_func_rc_t;
 
-
 /**
  * Function type for handling log messages.
  *
- * @param file     Source file name.
- * @param line     Source line number.
- * @param function Function name.
- * @param message  Log message - format string
- * @param ap       Log message format parameters.
+ * @param file      Source file name.
```

Comment:
plz, either set `.` at the end of each line, or do no use `.` at all

Reply:
done